### PR TITLE
net-misc/sshpass: version bump

### PR DESCRIPTION
**Pull request by laumann**
Signed-off-by: Thomas Bracht Laumann Jespersen <t@laumann.xyz>
Package-Manager: Portage-3.0.30, Repoman-3.0.3

### CODE REVIEW

File: net-misc/sshpass/sshpass-1.09.ebuild [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (thesamesam) [inline net-misc/sshpass/sshpass-1.09.ebuild:4]: EAPI 8?

File: net-misc/sshpass/sshpass-1.09.ebuild [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+DESCRIPTION="Tool for noninteractively performing password authentication with ssh"
+HOMEPAGE="https://sourceforge.net/projects/sshpass/"
+SRC_URI="mirror://sourceforge/${PN}/${PV}/${P}.tar.gz"
+LICENSE="GPL-2"
```
Comment (thesamesam) [inline net-misc/sshpass/sshpass-1.09.ebuild:9]: Put LICENSE in the next block, so it's DESCRIPTION/HOMEPAGE/SRC_URI (new blank line) LICENSE/SLOT/KEYWORDS, and [drop blank IUSE](https://projects.gentoo.org/pms/8/pms.pdf#section.7.3).

RDEPEND is OK as-is.

File: net-misc/sshpass/sshpass-1.09.ebuild (hunk 1 above)
Comment (laumann) [reply net-misc/sshpass/sshpass-1.09.ebuild]: Sorry, yes, done!

File: net-misc/sshpass/sshpass-1.09.ebuild (hunk 2 above)
Comment (laumann) [reply net-misc/sshpass/sshpass-1.09.ebuild]: Done. Thanks for the link.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**